components.py

```diff
@@ -156,36 +158,46 @@ def Block(title, *body, width="100%"): #width of the block in its container
     body : str, dbc
        Any number of elements which can be of various types to be
        formatted as a sentence within the card body.
-    width : str
+    ison : boolean, optional
+       Card is not displayed if False, default is True
+    width : str, optional
         html style attribute value to determine width of the card within its parent container.
+        Default `width` ="100%".
+
     Returns
     -------
     dbc.Card : component
        A dbc Card which has a pre-formatted title and body where the body can be any number of elements.
-       Default ` width='100%'`.
     """
+    if ison:
+        the_display = "inline-block"
+    else:
+        the_display = "none"
     return dbc.Card([
         dbc.CardHeader(title),
         dbc.CardBody(body),
-    ], className="mb-4 ml-4 mr-4", style={"display": "inline-block", "width": width})
+    ], className="mb-4 ml-4 mr-4", style={"display": the_display, "width": width})
```

Comment:
@kgraaf you've mentioned elsewhere that it's better to use CSS rather than in-line styles. Should we control visibility using a class instead of directly manipulating style here?